diff --git a/dist/drivers/fs.mjs b/dist/drivers/fs.mjs
index 6b463f5f6c960552ebe8ac901d38a94aba967e17..cd021cdc08c77efe5646a1b4781c518e270b754e 100644
--- a/dist/drivers/fs.mjs
+++ b/dist/drivers/fs.mjs
@@ -15,7 +15,7 @@ export default defineDriver((opts = {}) => {
       "**/.git/**"
     ];
   }
-  opts.base = resolve(opts.base);
+  /* opts.base = resolve(opts.base); */
   const r = (key) => {
     if (PATH_TRAVERSE_RE.test(key)) {
       throw new Error("[unstorage] [fs] Invalid key. It should not contain `..` segments: " + key);